### PR TITLE
Sometime there will be different types other than an object in params and those should ignore

### PR DESCRIPTION
**Pull request by spmsupun**
image

### CODE REVIEW

File: lib/openapi3.js [hunk 1]
```diff
@@ -179,6 +179,9 @@ function getParameters(data) {
     if (!Array.isArray(data.parameters)) data.parameters = [];
     data.longDescs = false;
     for (let param of data.parameters) {
+        if (typeof param !== 'object') {
+            continue;
```
Comment (spmsupun) [inline lib/openapi3.js:183]: Maybe we could do something like this?:
```suggestion
            param = {};
```

**Comment by MikeRalphson** [comment]
Can you give an example of where a param is not an object?

**Comment by MikeRalphson** [comment]
ping @spmsupun

**Comment by spmsupun** [comment]
@mikehardenize sorry I almost forgot about this. I don't think this need anymore

**Comment by mikehardenize** [comment]
You pinged the wrong mike

**Comment by spmsupun** [comment]
haha sorry

**Pull request closed without merging**